.github/workflows/push-docker-image.yaml

```diff
@@ -70,3 +70,17 @@ jobs:
           push: ${{ github.event_name != 'pull_request' }}
           tags: ${{ steps.meta.outputs.tags }}
           labels: ${{ steps.meta.outputs.labels }}
+
+      - name: Push To quay.io
+        id: push-to-quay
+        uses: redhat-actions/push-to-registry@v2
+        with:
+          context: .
```

Comment:
Warning: Unexpected input(s) 'context', 'file', 'platforms', 'push', 'labels', valid inputs are ['image', 'tags', 'registry', 'username', 'password', 'tls-verify', 'digestfile', 'extra-args']

The keyword `context` has no meaning here. 

This particular Github Action only supports `['image', 'tags', 'registry', 'username', 'password', 'tls-verify', 'digestfile', 'extra-args']`

Reply:
Did you mean to use `docker/build-push-action@v1`?